locale/fr_FR/user.xml

```diff
@@ -4,26 +4,50 @@
 <!--
   * locale/fr_FR/user.xml
   *
-  * Copyright (c) 2014-2018 Simon Fraser University
-  * Copyright (c) 2000-2018 John Willinsky
+  * Copyright (c) 2014-2019 Simon Fraser University
+  * Copyright (c) 2000-2019 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
   *
   * Localization strings for the fr_FR (Français (France)) locale.
   *
   -->
 
 <locale name="fr_FR" full_name="Français (France)">
+	<message key="user.accountInformation">Account Information</message>
 	<message key="user.affiliation">Affiliation</message>
-	<message key="user.affiliations">Affiliations</message>
-	<message key="user.biography">Note biographique</message>
+  <message key="user.affiliation.description">(Votre institution, par ex., "Université de Montréal")</message>
```

Comment:
@mpbraendle, please make sure your editor is set up to use tabs for indents rather than commas...